Ginger/GingerCoreNET/ActionsLib/ActCliOrchestration.cs

```diff
@@ -0,0 +1,194 @@
+﻿using GingerCore.Actions;
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Enums;
+using Amdocs.Ginger.Common.InterfacesLib;
+using Amdocs.Ginger.Repository;
+using Applitools.Utils;
+using GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static GingerCore.Actions.ActCLIOrchestration;
+using CliWrap;
+using DocumentFormat.OpenXml.Office.Word;
+using CliWrap.Buffered;
+using CliWrap.EventStream;
+using System.Text.RegularExpressions;
+using System.IO;
+using Applitools;
+using Microsoft.Azure.Cosmos.Serialization.HybridRow;
+using System.Diagnostics;
+using GingerExternal;
+
+namespace GingerCore.Actions
+{
+    public class ActCLIOrchestration : ActWithoutDriver
+    {
+        public override String ActionType
+        {
+            get
+            {
+                return "CLI Orchestration";
+            }
+        }
+
+        public override string ActionDescription { get { return "CLI Orchestration"; } }
+
+        public override bool ObjectLocatorConfigsNeeded { get { return false; } }
+
+        public override bool ValueConfigsNeeded { get { return true; } }
+
+        public override List<ePlatformType> Platforms
+        {
+            get
+            {
+                if (mPlatforms.Count == 0)
+                {
+                    AddAllPlatforms();
+                }
+                return mPlatforms;
+            }
+        }
+
+        public override string ActionEditPage { get { return "ActCLIOrchestrationEditPage"; } }
+
+        public override string ActionUserDescription { get { return "Perform File operations like Check if File Exists, Execute a file "; } }
+
+        public override void ActionUserRecommendedUseCase(ITextBoxFormatter TBH)
+        {
+            TBH.AddText("Use this action to perform File operations ");
+
+        }
+
+        public new static partial class Fields
+        {
+            public static string SourceFilePath = "SourceFilePath";
+            public static string FileOperationMode = "FileOperationMode";
+            public static string DestinationFolder = "DestinationFolder";
+        }
+
+        [IsSerializedForLocalRepository]
+        public string ScriptInterpreter { get; set; }
```

Comment:
Remove properties which are not needed for this action

Reply:
It is used Rename to FilePath